website/blog/2022-06-16-resources-migrating-your-react-native-library-to-the-new-architecture.md

```diff
@@ -0,0 +1,112 @@
+- **Leverage the Q&A [section](https://github.com/reactwg/react-native-new-architecture/discussions/categories/q-a) if you face a blocker and need support**. Our team and community experts are monitoring and will support at our best effort.
+- **Keep an eye on the other sections for topics that may affect you**. A new release may introduce exactly the API that you were looking for. You can subscribe to particular discussions via Github.
+
+We plan to support this group until the **New Architecture** is enabled by default and all the major libraries have been migrated to it.
+
+### Migration Status of Certain Libraries
+
+We'd like to give you a snapshot of popular React Native libraries and what's the status of their migration effort from the Working Group:
+
+- [react-native-gesture-handler](https://github.com/reactwg/react-native-new-architecture/discussions/15): ✅ Migrated
+- [react-native-navigation](https://github.com/reactwg/react-native-new-architecture/discussions/17): 🏃‍♂️ Ongoing
+- [react-native-pager-view](https://github.com/reactwg/react-native-new-architecture/discussions/16): 🏃‍♂️ Ongoing
+- [react-native-reanimated](https://github.com/reactwg/react-native-new-architecture/discussions/14): ✅ Migrated. In the process of testing and profiling for performances
+- [react-native-screens](https://github.com/reactwg/react-native-new-architecture/discussions/13): 🏃‍♂️ Ongoing
+- [react-native-slider](https://github.com/reactwg/react-native-new-architecture/discussions/38): 🎬 Started
+- [react-native-template-new-architecture](https://github.com/reactwg/react-native-new-architecture/discussions/21): ✅ Migrated. Gradually adopting/testing more companion Libraries
+- [react-native-template-typescript](https://github.com/reactwg/react-native-new-architecture/discussions/22): ✅ Migrated
+- [react-native-webview](https://github.com/reactwg/react-native-new-architecture/discussions/19): 🎬 Started
+
+## Next Steps
+
+We are invested in supporting the React Native community’s adoption of the new architecture. Concretely, we will continue to:
+
+- Offer best-effort support in the **Working Group.**
+- Provide more examples about how to achieve amazing results with the new architecture in the **RNNewArchitecture** repositories.
+- Provide clear and up-to-date documentation on the **New Architecture**.
+- Track the migration status of essential React Native libraries in the **Working Group**.
+- Simplify the migration path for developers
+
+In addition, React Native 0.69 will ship with improved devX for app and library developers for new architecture adoption. You can find more information about the 0.69.0 release [here](https://github.com/reactwg/react-native-releases/discussions/21).
```

Comment:
This phrase imply this blogpost will come out before 0.69.0 - @cortinico @fortmarek we should probably revise the blogpost draft to account for this blogpost existing (maybe?)

Personally I think we should avoid publishing two blogposts back-to-back, it makes it more likely that folks will miss out on one or the other; so if we want them to go out in the span of a few days, let's make sure that the other one has a link to the previous to raise awareness

Reply:
Thanks for the callout @kelset - it'd be great to link to this from the 0.69 blogpost. If we could merge this by Monday and then ~ Wednesday release 0.69, that'd be a reasonable schedule imho. @cipolleschi wdyt? I don't want to rush this if you think this needs more time.

Reply:
Yes, sure! 

I'm apply all the comments and suggestions right now. On Monday I'll be on vacation (I had already planned to try and publish this ASAP!), but if everything is right, feel free to push the merge button!